### PR TITLE
New Canary documentation

### PR DESCRIPTION
**Pull request by joaopapereira**

### CODE REVIEW

File: deploy-apps/canary-deploy.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+
+For more information about CAPI, see the
+[Cloud Foundry API (CAPI) documentation](https://v3-apidocs.cloudfoundry.org/).
+
+Canary deployments will allow the users to deploy a new version of an application and check if it is running as expected,
+afterwards continue the rolling deployment of the rest of the instances.
+
+The expected workflow for this feature is:
+1. Deploy a single canary instance with the changed code
+1. The user will manually validate that the instance is running as expected
```
Comment (a-b) [inline deploy-apps/canary-deploy.html.md.erb:20]: suggestion: `Manually verify that canary instance with new code runs as expected`

File: deploy-apps/canary-deploy.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+You can use Cloud Foundry Command Line Interface (cf CLI) commands
+or the Cloud Foundry API (CAPI) to push your apps to <%= vars.app_runtime_first %> using a canary deployment.
+
+For information about the traditional method for addressing app downtime while pushing app updates,
+see [Using blue-green deployment to reduce downtime and risk](./blue-green.html).
+
+For more information about CAPI, see the
+[Cloud Foundry API (CAPI) documentation](https://v3-apidocs.cloudfoundry.org/).
+
+Canary deployments will allow the users to deploy a new version of an application and check if it is running as expected,
```
Comment (Samze) [inline deploy-apps/canary-deploy.html.md.erb:15]:
```suggestion
Canary deployments will allow users to deploy a new version of an application and check if it is running as expected,
```

File: deploy-apps/canary-deploy.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+afterwards continue the rolling deployment of the rest of the instances.
```
Comment (Samze) [inline deploy-apps/canary-deploy.html.md.erb:16]:
```suggestion
before continuing the rolling deployment of the rest of the instances. If the new version of the application is not running as expected, the canary deployment can be cancelled.
```

**Review by Samze — APPROVED** [review]

**Comment by pspinrad** [comment]
@joaopapereira @Samze will this also apply to TAS 7.0 / TZ-CF 10 and be in upcoming TAS 4, 5, and 6 patch versions?
If yes re TAS patch versions, how can I track which ones?
Thanks!

**Comment by joaopapereira** [comment]
We will need this for tas 6 as well

**Pull request merged**